cpp/src/groupby/hash/groupby.cu

```diff
@@ -58,7 +59,8 @@ namespace {
 constexpr std::array<aggregation::Kind, 7> hash_aggregations{
     aggregation::SUM, aggregation::MIN, aggregation::MAX,
     aggregation::COUNT_VALID, aggregation::COUNT_ALL,
-    aggregation::ARGMIN, aggregation::ARGMAX};
+    aggregation::ARGMIN, aggregation::ARGMAX,
+    aggregation::MEAN};
```

Comment:
@devavret has thought more about additional compound aggs, but I believe things like `stddev` and `kurtosis` should be supported by the same infrastructure.

Reply:
Basically all aggregations that do not depend on the order of the values and return a single value should be able to compute using the hash method. I planned a method similar to sort groupby to call dependency aggregations. But this would need a different way to request dependency **single pass** aggregations like `sum` and `count`. So, sort of a hybrid.